src/goto-programs/goto_contractor.h

```diff
@@ -236,22 +250,21 @@ class Contractors
 class CspMap
 {
 public:
-  static constexpr int MAX_VAR = 26;
+  static constexpr int MAX_VAR = 20;
```

Comment:
I still don't understand this limit, why we have it or why it would need to be lowered. Could you please explain it?

Reply:
This represents the variables for a given constraint. I reached this number because it is the maximum number of variables in a single condition in SV-Comp. However, I want to explore the possibility of having it dynamic if you have a suggestion.

Reply:
Could you please add your explanation as a comment, together with a TODO saying something like "this should be dynamic"?